models/repository/repository.go

```diff
@@ -45,3 +79,22 @@ func (u *RepositoryStruct) Save() bool {
 	u.ID, _ = result.LastInsertId()
 	return true
 }
+
+func (u *RepositoryStruct) Authenticate(token string, response []byte) bool {
```

Comment:
__[golint]__
* [golint] exported method RepositoryStruct.Authenticate should have comment or be unexported